core/src/main/scala/io/finch/Endpoint.scala

```diff
@@ -696,12 +693,9 @@ object Endpoint {
     *     ...
     * }}}
     */
-  def filesystemAsset[F[_]](path: String)(implicit
-      F: Sync[F],
-      S: ContextShift[F]
-  ): Endpoint[F, Buf] = {
+  def filesystemAsset[F[_]](path: String, blockingEc: ExecutionContext)(implicit F: Async[F]): Endpoint[F, Buf] = {
```

Comment:
Should we take implicit EC?

Reply:
`Async` already has a blocking EC baked in

Reply:
@joroKr21 thanks, completly missed it!